client/post-editor/editor-categories-tags/accordion.jsx

```diff
@@ -163,27 +140,28 @@ module.exports = React.createClass( {
 			subtitlePieces.push( tagsSubtitle );
 		}
 
-		if ( ! isPage( this.props.post ) && this.isCategoriesLoading() ) {
-			return this.translate( 'Loading…' );
-		}
-
 		return subtitlePieces.join( ', ' );
-	},
+	}
 
-	getTitle: function() {
+	getTitle() {
+		const { translate, postType } = this.props;
 		let title;
-		if ( isPage( this.props.post ) ) {
-			title = this.translate( 'Tags' );
+		if ( postType === 'page' ) {
+			title = translate( 'Tags' );
 		} else {
-			title = this.translate( 'Categories & Tags' );
+			title = translate( 'Categories & Tags' );
 		}
 		return title;
-	},
+	}
 
-	render: function() {
-		var classes = classNames( 'editor-drawer__accordion', 'editor-categories-tags__accordion', this.props.className, {
-			'is-loading': ! this.props.site || ! this.props.post || this.isCategoriesLoading()
-		} );
+	render() {
+		const classes = classNames(
+			'editor-drawer__accordion',
+			'editor-categories-tags__accordion',
+			this.props.className, {
+				'is-loading': this.isLoading()
```

Comment:
Looks like the purpose of `is-loading` is to apply the placeholder styles to the subtitle:

https://github.com/Automattic/wp-calypso/blob/57f3de7/client/post-editor/editor-drawer/style.scss#L28-L41

Since we don't render a subtitle while loading, we can probably get rid of this class modifier.